ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource

```diff
@@ -175,7 +175,7 @@ Set Model Server Name
 
 Enable External Serving Route
     [Documentation]    Enables the serving route to be accessed externally
-    SeleniumLibrary.Select Checkbox    xpath://input[@id="alt-form-checkbox-route"]
+    SeleniumLibrary.Select Checkbox    xpath://input[@id="alt-form-checkbox-auth"]
```

Comment:
same comment https://github.com/red-hat-data-services/ods-ci/pull/1735/files#r1726552585

Reply:
@bdattoma Yeah i have verified . Its route only. made the changes

Reply:
> Its route only. 


what do you mean?


> made the changes


what changes?

Reply:
While working on the automation using the 2.11 cluster, I noticed that it doesn't have the model route option, which caused an error. To resolve this, I changed the XPath from route to auth and raised a PR. After your comment, I checked with the 2.12 cluster, which does have the model route option. When I ran my automation on the 2.12 cluster, it threw an error because of the XPath change. Since the model route option is present in 2.12, I reverted the XPath change from auth back to route.

Reply:
> Since the model route option is present in 2.12, I reverted the XPath change from auth back to route.


hm I still see the `auth` one. Maybe you didn't push the change?

Reply:
yes working on other changes aswell will push it